.github/workflows/ci.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           - operating-system: 'macos-latest'
             php-version: '8.2'
             job-description: 'on macOS'
+            FAST_LINT_TEST_CASES: 1
```

Comment:
overall +1, but pls add comment to both windows and macos build why we set up FAST_LINT_TEST_CASES for them